src/telescope_baseline/mapping/__init__.py

```diff
@@ -0,0 +1,8 @@
+__all__ = []
+
+__version__ = '0.1'
+__uri__ = ''
+__author__ = 'JASMINE e2e group'
+__email__ = ''
+__license__ = ''
+__description__ = ''
```

Comment:
Should we define these items in each subdirectory?
Defining them on the top level seems sufficient.

Reply:
thanks, deleted